cpp/src/Ice/SSL/SSLException.cpp

```diff
@@ -2,7 +2,7 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
 
-#include "Ice/SSL/Certificate.h"
+#include "Ice/SSL/Exception.h"
```

Comment:
I would prefer Exception.{h,cpp} or SSLException.{h,cpp}.

Reply:
I will add SSL prefix to the header, we want unique names for the source files.